tap_shiphero/client.py

```diff
@@ -63,7 +67,21 @@ def request(self, method, path, **kwargs):
 
         response.raise_for_status()
 
-        return response.json()
+        try:
+            return response.json()
+        except:
+            LOGGER.info('----------------------------------------')
+            # I want as much information here as possible, but I really
+            # only care about the `content-type` here
+            LOGGER.info('response.headers: ', response.headers)
+            # Here I want to see what is coming back
+            LOGGER.info('response.content: ', response.content)
```

Comment:
This could include data, PII, etc. Content should not be logged, only through local testing.